tests/unit/api/main.py

```diff
@@ -572,7 +578,7 @@ def test_node_connect_network_no_such_node(self):
         network_create_simple('hammernet', 'anvil-nextgen')
 
         with pytest.raises(api.NotFoundError):
-            api.node_connect_network('node-98', '99-eth0', 'hammernet') # changed
+            api.node_connect_network('node-98', '99-eth0', 'hammernet')  # changed # noqa
```

Comment:
what about move the ``#changed`` to right next line to the `` with pytest.raises(api.NotFoundError):`` ?